en/peripherals/companion_computer_peripherals.md

```diff
@@ -44,10 +44,36 @@ Cameras are among the most common form of data collection on a drone whether it
 
 ### Stereo Cameras
 
-Stereo cameras are typically used for depth perception, path planning and SLAM. They are in no way guaranteed to be plug and play with your companion computer.
+Stereo cameras are typically used for depth perception, path planning and SLAM. 
+They are in no way guaranteed to be plug and play with your companion computer.
 
 #### Where To Buy
 * [Intel® RealSense™ Depth Camera D435](https://click.intel.com/intelr-realsensetm-depth-camera-d435.html)
 * [Intel® RealSense™ Depth Camera D415](https://click.intel.com/intelr-realsensetm-depth-camera-d415.html)
 * [DUO MLX ](https://duo3d.com/product/duo-minilx-lv1)
 * [Ironsides](https://www.perceptin.io/products)
+
+
+## Data Telephony (LTE) {#data_telephony}
+
+An LTE USB module can be attached to a companion computer and used to route MAVLink traffic between the flight controller and the Internet.
+
+There is no "standard method" for a ground station and companion to connect over the Internet.
+Generally you can't connect them directly because neither of them will have a public/static IP on the Internet.
+
+> **Note** Typically your router (or the mobile network) has a public IP address, and your GCS computer/vehicle are on a *local* network.
+  The router uses network address translation (NAT) to map *outgoing* requests from your local network to the Internet, and can use the map to route the *responses* back to requesting system.
+  However NAT has no way to know where to direct the traffic from an arbitrary external system, so there is no way to *initiate* a connection to a GCS or vehicle running in the local network.
+
+A common approach is to set up a virtual private network between the companion and GCS computer (i.e. install a VPN system like [zerotier](https://www.zerotier.com/) on both computers).
+The companion then uses [mavlink-router](https://github.com/intel/mavlink-router) to route traffic between the serial interface (flight controller) and GCS computer on the VPN network.
+
+This method has the benefit that the GCS computer address can be static within the VPN, so the configuration of the *mavlink router* does not need to change over time.
```

Comment:
Since we are talking about sending data over the internet here, I find it important to mention security as well. I.e. MAVLink does not use encryption, but the VPN gives you a secure link.

Reply:
OK, added